cmd/pmm-configurator/instance.go

```diff
@@ -43,6 +43,9 @@ func checkInstance(instanceID string) (string, error) {
 		if err != nil {
 			return "Cannot fetch instance meta-data", err
 		}
+		if resp.StatusCode == 404 {
+			return "success", nil
```

Comment:
"page not found" is considered "success"? It's fine, but if there is no trivial logic, could you write down some comments? It's not clear to me what really this `checkInstance` function does and why `404` in case of this request is considered "success". No need to explain me here, it's just a suggestion to be more verbose/clear in the code itself.

Second thing. It's kind of anti pattern to depend on string as a result of func.
```
	result, err := checkInstance(passedInstance.ID)
	if result == "success" {
		returnSuccess(w)
	} else {
		returnError(w, req, http.StatusForbidden, result, err)
}
```
why not:
```
	result, err := checkInstance(passedInstance.ID)
	if err != nil {
		returnError(w, req, http.StatusForbidden, result, err)
	}
	returnSuccess(w)
```
It's a statically typed language so why depend on dynamic string.

Reply:
1) yep, the comment should be added
2) simplified